UbuconAsiaCodeOfConduct.md

```diff
@@ -5,6 +5,8 @@ The goal of Ubucon Asia, as a part of the greater Ubuntu Community, is to connec
 
 Both enforced to anyone participating Ubucon Asia event. All participants are expected to cooperate to ensure a safe environment for everybody.
 
+This code of conduct applies to all communication: this includes virtual communication such as mailing list, eventyay.com, github.com and other communication medium such as social media platform and physical meetup
```

Comment:
I think this sentence is similar to one in the `A safe environment where everyone is respected` section.

>Sexual language and imagery is not appropriate for anywhere, anytime of the conference including every sessions and any online spaces.